### ISSUE TITLE
Guest user

### ISSUE THREAD

**Issue by meanin** [meanin/candidate-evaluator#49]
The repository should hold user email address and data passed in the registration form. Furthermore, the controller (Register method) and cqs/cqrs classes are needed as well.

The output of this task should be the whole workflow for adding a new guest user to the repository from top to bottom.

**Comment by meanin** [comment]
No, a person who would like to test our app for a given amount od time :)

**Comment by devmonte** [comment]
Aaa, you mean free access for 30 days :D
